es6-destructuring/main.js

```diff
@@ -0,0 +1,56 @@
+// Object Destructuring
+
+const book1 = {
+  title: 'Goodnight Punpun',
+  author: 'Inio Asano',
+  libraryID: 3353
+};
+
+const { title: title1, author: author1, libraryID: library1 } = book1;
```

Comment:
The instructions say to use the same names as the fields. That lets you use the shorthand syntax.